### PR TITLE
not sticky yet

### PR DESCRIPTION
**Pull request by billyfung**
As per #63, haven't made the footer sticky yet though

### CODE REVIEW

**Comment by jinpark** [comment]
this might helpz https://github.com/Semantic-Org/Semantic-UI/issues/2882

**Comment by jinpark** [comment]
I did some things @billyfung 

see if its lgtm

**Comment by billyfung** [comment]
screen shot 2016-02-03 at 5 34 57 pm
Safari problem

**Comment by jinpark** [comment]
i restikied for safari

**Comment by billyfung** [comment]
ok works fine now, but will probably change colour before merging

File: templates/index.html [hunk 1]
```diff
@@ -91,6 +91,15 @@ <h2 class="active item">Tell Me About Coffee</h2>
     </div>
 </div>
 
+  <footer class="ui inverted grey footer segment">
+    <div class="ui center aligned container">
+      <div class="ui horizontal inverted small divided link list">
+        <a class="item" href='mailto:billy@billyfung.com?Subject=tellmeabout.coffee'>Contact</a>
+        <div class="item">By <a href='billyfung.com'>Billy</a> and <a href='jinpark.net'>Jin</a></div>
```
Comment (jinpark) [inline templates/index.html]: Have to make these absolute urls or theyll go to `tellmeabout.coffee.com/billyfung.com`

File: templates/index.html [hunk 2]
```diff
@@ -91,6 +91,15 @@ <h2 class="active item">Tell Me About Coffee</h2>
     </div>
 </div>
 
+  <footer class="ui inverted grey footer segment">
+    <div class="ui center aligned container">
+      <div class="ui horizontal inverted small divided link list">
+        <a class="item" href='mailto:billy@billyfung.com?Subject=tellmeabout.coffee'>Contact</a>
+        <div class="item">By <a href='http://billyfung.com'>Billy</a> and <a href='http://jinpark.net'>Jin</a></div>
```
Comment (jinpark) [inline templates/index.html]: Can you also `target="_blank" for these so they open in a new tab?
http://www.w3schools.com/tags/att_a_target.asp

**Comment by jinpark** [comment]
lgtm
good job mr pleb

**Pull request merged**